ansible/roles/debops.apt_mark/files/script/apt-mark-status

```diff
@@ -13,7 +13,7 @@ def run_command(command_string):
                     shell=True, preexec_fn=os.setsid)
     stdout, stderr = process.communicate()
     if stdout:
-        return stdout.strip().split('\n')
+        return stdout.decode('utf-8').strip().split('\n')
```

Comment:
This will fail when the user's default locale is not UTF-8. I don't think there is a good reason for the user to change it to something else, but still.

Reply:
@bfabio This seems to me as a go-to way to handle strings in code compatible with both Python 2.7 and Python 3.x, do you know a better way? Python 3 is supposed to use Unicode (not sure if specifically UTF-8) by default.

Reply:
@drybjed We can use `Popen([command_string], ..., env=dict(os.environ, LANG="C")`) and `stdout.decode('ascii')`.